app/controllers/admin/schedules_controller.rb

```diff
@@ -30,12 +30,19 @@ def index
 
     def create
       invoke_callbacks(:create, :before)
+
+      if params[:order_cycle_ids].blank?
```

Comment:
is this new behavior? we dont create a schedule without OCs.

Reply:
Nope. It was perfectly covered by

https://github.com/openfoodfoundation/openfoodnetwork/blob/c73defa8a1eaca230f58196058c8ae4e5bfec827/spec/controllers/admin/schedules_controller_spec.rb#L133-L137

I wouldn't have noticed otherwise, tbh. If you think about the UI it makes sense.

![Screenshot from 2020-05-29 15-21-25](https://user-images.githubusercontent.com/762088/83264269-21bc4200-a1c0-11ea-8632-7db3aea2246d.png)


@oeoeaio good a good job in terms of code coverage :ok_hand:

Reply:
:+1: